polaris-react/src/components/AppProvider/AppProvider.tsx

```diff
@@ -108,6 +108,19 @@ export class AppProvider extends Component<AppProviderProps, State> {
       this.stickyManager.setContainer(document);
       this.setBodyStyles();
       this.setRootAttributes();
+
+      const isSafari16 =
+        navigator.userAgent.includes('Safari') &&
+        !navigator.userAgent.includes('Chrome') &&
+        (navigator.userAgent.includes('Version/16.1') ||
+          navigator.userAgent.includes('Version/16.2') ||
+          navigator.userAgent.includes('Version/16.3'));
```

Comment:
Yes, to my understanding 👍

Reply:
Reference links:

[Blog post](https://clagnut.com/blog/2423)

> Update: as of Safari 16.4, this bug [has been fixed](https://github.com/WebKit/WebKit/pull/10126).

[WebKit PR](https://github.com/WebKit/WebKit/pull/10126)

> also means we are going
fast enough that we can enable optical sizing on every font - because doing so is just an
attribute in a dictionary, rather than creating a whole new CTFont. That is the fix for this
bug: enabling optical sizing on all fonts.